widget/ui/src/components/RefreshProgressButton/RefreshProgressButton.tsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable @typescript-eslint/no-magic-numbers */
```

Comment:
can you remove this and create constants for each number?

Reply:
fixed.